### ISSUE TITLE
[New API] Google Hotels Listing Autocomplete API

### ISSUE THREAD

**Issue by martin-serpapi** [serpapi/public-roadmap#1740]
A user requested that we scrape prices for different dates for a specific hotel. This seems to be available in the date picker of the `Prices` section for a listing:

![image](https://github.com/serpapi/public-roadmap/assets/144151895/36d4fc30-9538-4c22-836d-074b5cd4bda2)

[Google Hotels](https://www.google.com/travel/search?q=hotels%20barcelona&qs=CAAgASgAMiZDaGdJdDVtOW9OTEhucFA5QVJvTEwyY3ZNWFJ1TVhsck1IY1FBUTgNSAA&ts=CAESCgoCCAMKAggDEAAaVAo0EjIyJTB4MTJhNDk4MTY3MThlMzBlNToweDQ0YjBmYjNkNGY0NzY2MGE6CUJhcmNlbG9uYRIcEhQKBwjoDxAHGBYSBwjoDxAHGBcYATIECAAQACoNCgk6A0JHTmICCAEaAA&ap=KigKEgm4feKL75xEQBE-Rx0dY2MAQBISCd_jTE1sykRAET5HHR1L4wFAMAC6AQZwcmljZXM)

[Front](https://app.frontapp.com/open/cnv_1bjn50yk?key=22Q3-I8HeinWqgWMtCgbJYLIRY5wWZu2)

**Comment by marm123** [comment]
A potential Enterprise Customer requested this.

Moving this to `queued`

[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/215474987440774?view=List)

**Comment by hilmanski** [comment]
@marm123 Should we update the title? I feel like the title is a bit confusing. Maybe `Calendar Prices` ?

**Comment by marm123** [comment]
@hilmanski yeah, that's much better. Though I think Google Hotels Price Calendar API is better.
